mubench.pipeline/tests/data/test_detector.py

```diff
@@ -96,9 +96,21 @@ def test_uses_first_release_by_default(self):
         assert_equals(expected_url, detector.jar_url)
         assert_equals("-md5_requested-", detector.md5)
 
+    def test_uses_first_cli_version_for_debug_release(self):
+        self.setup_releases([
+            {"md5": "-md5_0-", "tag": "-release_0-", "cli_version": RunnerInterfaceTestImpl.TEST_VERSION},
+            {"md5": "-md5_1-", "tag": "-release_1-", "cli_version": "0.-1"}])
```

Comment:
I was a little surprised by this. But I guess the easiest way to tell the pipeline which CLI version to use is indeed to have a `release.yml` in the directory with the detector Jar that I want to debug... which brings me to the crucial point: we need to update the documentation ;)

Reply:
Let's try to generate and mount a `release.yml` with a CLI version specified on the command line, to avoid any special handling of this case.